pkgs/development/python-modules/Cython/default.nix

```diff
@@ -30,10 +30,12 @@ buildPythonPackage rec {
       tests/run/cpdef_enums.pyx
   '';
 
-  buildInputs = [ glibcLocales pkgconfig gdb ];
-  # For testing
-  nativeBuildInputs = [ numpy ncurses ];
-
+  nativeBuildInputs = [
+    pkgconfig
+    # For testing
+    numpy ncurses
```

Comment:
Do these actually need to be native?

Reply:
Probably not. Did this come from a minimal edit?

Reply:
Yeah it did. @FRidh I agree it looks weird, but probably better to fix separately. Because this one is so big, I want to keep it boring.